test/lib/Elastica/Test/Query/MultiMatchTest.php

```diff
@@ -83,6 +83,14 @@ public function testFuzzy()
         $resultSet = $this->_getResults($multiMatch);
 
         $this->assertEquals(0, $resultSet->count());
+
+        $multiMatch = new MultiMatch();
+        $multiMatch->setQuery('Tritsan'); // Misspell on purpose
+        $multiMatch->setFields(array('full_name', 'name'));
+        $multiMatch->setFuzziness('AUTO');
```

Comment:
@JustinHook I would suggest we make a "constant" out of this which can be used. What do you think?